libraries/AP_BoardConfig/AP_BoardConfig.cpp

```diff
@@ -97,6 +102,16 @@ const AP_Param::GroupInfo AP_BoardConfig::var_info[] = {
     AP_GROUPINFO("CAN_ENABLE", 6, AP_BoardConfig, _can_enable, 0),
 #endif
     
+#if CONFIG_HAL_BOARD == HAL_BOARD_PX4
```

Comment:
I would prefer it separate, but it's not a strong "belief". I don't know what is usually used in ArduPilot, that would be the preferred.
Either way, it's your PR so, unless contradicting any ArduPilot style, it should go in with your preferred style.